homeassistant/components/broadlink/updater.py

```diff
@@ -31,6 +31,7 @@ def get_update_manager(device):
         "RM4": BroadlinkRMUpdateManager,
         "SP1": BroadlinkSP1UpdateManager,
         "SP2": BroadlinkSP2UpdateManager,
+        "BG1": BroadlinkBG1UpdateManager,
```

Comment:
Move the "BG1" class to the second position.

Reply:
Done